### PR TITLE
cloak: init at 0.2.0

### PR DESCRIPTION
**Pull request by illdefined**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 139296` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cloak</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/139296#issuecomment-926470667)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-139296]$ ./results/cloak/bin/cloak help
cloak 0.2.0
Evans Murithi <murithievans80@gmail.com>
A Command Line OTP (One Time Password) Authenticator application.
[...]
```

Looks good to me.

File: pkgs/applications/misc/cloak/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    owner = "evansmurithi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "139z2ga0q7a0vwfnn5hpzsz5yrrrr7rgyd32yvfj5sxiywkmczs7";
+  };
+
+  cargoSha256 = "0af38wgwmsamnx63dwfm2nrkd8wmky3ai7zwy0knmifgkn4b7yyj";
+
+  meta = with lib; {
+    homepage = "https://github.com/evansmurithi/${pname}";
```
[the remark this answers is not in the document]
Comment (illdefined) [reply pkgs/applications/misc/cloak/default.nix]: Modified as suggested.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 139296` at e3f9796d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>cloak</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/applications/misc/cloak/default.nix:1:8:
```
  |
1 | { lib, stdenv, rustPlatform, fetchFromGitHub }:
  |        ^
```
</li>
  </ul>
</details>

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 139296` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cloak</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]

**Pull request merged**